packages/website/docs/FlatSelectConnected.md

```diff
@@ -0,0 +1,27 @@
+# Best Practices
+The Flat select option should be used for single choice selection.
+
+The width of each of the options should be the same, that way we keep everything equal and balanced, we don't want any of the options feel more important than the other.
```

Comment:
It's just my opinion, but I find this part of the sentence doesn't add any value.

```suggestion
The width of each of the options should be the same, we don't want any of the options feel more important than the other.
```